### DISCUSSION TITLE
A figure that presents the uncertainty

### DISCUSSION THREAD

**Discussion by mmikhasenko** [ComPWA/polarimetry#130] [Ideas]
I thought of a way to plot the uncertainty. That is just an idea, let's discuss if it is worth and how to do in details.

The uncertainty of \vec \alpha are two kind:
 - length of the arrow
 - direction of the arrow.

The direction is a coordinate in a solid angle (\theta,\phi). The full solid angle is 4pi, thefore, the relative uncertainty is
```
delta Omega = (the area of the solid angle) / 4pi
```

The question now is how to compute the area. In case of Cartesian coordinates, I would talk (mix, max) for all dimensions, having a box uncertainty with the area, `(max1-min1) * (max2-min2)`.
Alternatively, one can draw an ellipse that covers all systematics points.

Finding the minimal area (box || ellipse) that covers all syst. points in spherical coordinates is a fun problem.
I do not have an immediate solution.

**Comment by redeboer** [comment]
Moved to https://github.com/redeboer/polarization-sensitivity/issues/139
